packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -114,8 +115,61 @@ describe('RestServer (integration)', () => {
       .expect(200, content);
   });
 
+  it('allows static assets to be mounted on multiple paths', async () => {
+    const root = ASSETS;
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+      },
+    });
+
+    server.static('/html-0', root);
+    server.static('/html-1', root);
+
+    const content = fs
+      .readFileSync(path.join(root, 'index.html'))
+      .toString('utf-8');
+    await createClientForHandler(server.requestHandler)
+      .get('/html-0/index.html')
+      .expect('Content-Type', /text\/html/)
+      .expect(200, content);
+    await createClientForHandler(server.requestHandler)
+      .get('/html-1/index.html')
+      .expect('Content-Type', /text\/html/)
+      .expect(200, content);
+  });
+
+  it('allows static assets to be mounted on the same path multiple times', async () => {
+    const root = ASSETS;
+    const otherAssets = path.join(FIXTURES, 'other-assets');
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+      },
+    });
+
+    server.static('/html', root);
+    server.static('/html', otherAssets);
+
+    let content = fs
+      .readFileSync(path.join(root, 'index.html'))
+      .toString('utf-8');
```

Comment:
This can be simplified, see https://nodejs.org/docs/latest-v8.x/api/fs.html#fs_fs_readfilesync_path_options

```ts
let content = fs.readFileSync(path.join(root, 'index.html'), {encoding: 'utf-8'});
```

It would be even better to extract a shared helper function to avoid repetition.

```ts
let content = await readFileFromDirectory(root, 'index.html');

// impl
const readFileAsync = util.promisify(fs.readFile);
function readFileFromDirectory(dirname: string, filename: string): Promise<string> {
  return readFileAsync(path.join(dirname, filename), {encoding: 'utf-8'});
}
```